featuretools/entityset/entityset.py

```diff
@@ -173,6 +173,21 @@ def __deepcopy__(self, memo):
     def dataframes(self):
         return list(self.dataframe_dict.values())
 
+    @property
+    def dataframe_type(self):
+        '''String specifying the library used for the entity dataframes. Null if no entities'''
+        df_type = None
+
+        if self.entities:
+            if isinstance(self.entities[0].df, pd.DataFrame):
+                df_type = Library.PANDAS.value
+            elif isinstance(self.entities[0].df, dd.DataFrame):
+                df_type = Library.DASK.value
+            elif is_instance(self.entities[0].df, ks, 'DataFrame'):
+                df_type = Library.KOALAS.value
```

Comment:
can you update this property to use the `self.dataframes` attribute instead of `self.entities`?

Reply:
Yeah updated in 60350df50306d62acf3dbfea615986238a827845